### DISCUSSION TITLE
分享一个详细的绑卡教程

### DISCUSSION THREAD

**Discussion by woodchen-ink** [ChatGPTNextWeb/NextChat#1419] [Show and tell]
教程： https://woodchen.ink/1241.html

**Comment by leochen12-rgb** [comment]
Nobepay、Depay动不动就维护，还是算了吧

**Reply by woodchen-ink** [reply]
> Nobepay、Depay动不动就维护，还是算了吧

是吗，我还真不知道，我才用了半个多月，还没碰到维护的，绑卡完成后每个月登录付下钱不就好了😂，又不用经常登
